merino-suggest/src/lib.rs

```diff
@@ -68,6 +69,8 @@ pub struct SuggestionRequest {
     /// The user agent of the request, including OS family, device form factor, and major Firefox
     /// version number.
     pub device_info: DeviceInfo,
+    /// The id for the request
+    pub request_id: Uuid,
```

Comment:
Can you talk a bit about why you added this to the `SuggestionRequest`, instead of having the `merino-web` view extract it independently? My intuition was this could be only a part of the web crate, but I haven't thought through it too much yet.